tests/config/ray-cluster.ray-ft.yaml.template

```diff
@@ -92,6 +92,8 @@ spec:
               # RAY_REDIS_ADDRESS can force ray to use external redis
               - name: RAY_REDIS_ADDRESS
                 value: redis:6379
+              - name: RAY_gcs_rpc_server_reconnect_timeout_s
+                value: "20"
```

Comment:
Do we intend to check this value somewhere? I didn't see it in the test

Reply:
This is used to accelerate the test. Some tests in `compatibility-test.py` will kill the GCS server process on the head Pod and wait until the new one is available. By default, the head will crash after 60 seconds. With this configuration, it will crash after 20 seconds.

Reply:
Oh right. This is another place where a code comment would be helpful

Reply:
Added comment in https://github.com/ray-project/kuberay/pull/1036/commits/d0991ff50ac8437ed24517c4fa9c797add3349ca